pkg/services/ngalert/services/alertmanager_config.go

```diff
@@ -0,0 +1,237 @@
+package services
```

Comment:
I think `notifier` is the second best option. In my other PR, we renamed `services` -> `provisioning`, but this new file isn't provisioning-related, so it needed a new home. I think since the AM logic is already in notifier, that's probably the most relevant place for this one-off file for the time being.

`pkg/services/ngalert/services` definitely did feel weird.

Reply:
> I think notifier is the second best option

I also agree with `notifier` being the right place.